docs/crocks/Either.md

```diff
@@ -0,0 +1,89 @@
+# Either
+
+`Either a b`
+
+Crock which --
+
+The "`Either`" crock creates disjunction with a linear data flow.
+
+```js
+--
+```
+
+`Either` exposes these constructors and instances:
+
+| Constructor | Instance |
+|:---|:---|
+| [`Left`](#left), [`Right`](#right), [`of`](#of) | [`inspect`](#inspect), [`value`](#value), [`type`](#type), [`equals`](#equals), [`map`](#map), [`ap`](#ap), [`of`](#of), [`chain`](#chain), [`sequence`](#sequence), [`traverse`](#traverse) |
+
+## Constructors
+
+### Left
+
+`Either m => a -> m a _`
```

Comment:
Would be nice to keep the `m c a` for 🌽sistancy.

So in this case, it may make sense to so something like
`Either m => c -> m c _`